Source/Fuse.Controls.ScrollView/Triggers/ScrollingAnimation.uno

```diff
@@ -93,6 +93,8 @@ namespace Fuse.Triggers
 				_hasFrom = true;
 				if (!_hasRange)
 					_range = ScrollingAnimationRange.Explicit;
+
+				BypassSeek(OffsetScrollProgress);
```

Comment:
This needs a `if (_scrollable != null)`, which can only be true if the control is rooted and has a scrollview.